pkg/controller/kubemigrator.go

```
@@ -120,11 +121,13 @@ func (km *KubeMigrator) processOne(obj interface{}) error {
 	if err == nil {
 		_, err = km.updateStatus(m, migrationv1alpha1.MigrationSucceeded, "")
 		metrics.Metrics.ObserveSucceededMigration(resource(m).String())
+		klog.V(2).Infof("%v: migration succeeded", m.Name)
 		return err
 	}
-	_, err = km.updateStatus(m, migrationv1alpha1.MigrationFailed, err.Error())
+	_, updateErr := km.updateStatus(m, migrationv1alpha1.MigrationFailed, err.Error())
 	metrics.Metrics.ObserveFailedMigration(resource(m).String())
-	return err
+	klog.Errorf("%v: migration failed: %v", m.Name, err)
+	return updateErr
```

thi sis a change in behavior. Why shouldn't we return the failure when we failed.  This allows us to return nil after having an error.  If this is correct, add comment explaining why.

---

Did not intend to change behavior. Any recommendations for handling the `updateErr` ? 
Please comment at https://github.com/kubernetes-sigs/kube-storage-version-migrator/pull/58 instead.